llvm/unittests/Object/ELFObjectFileTest.cpp

```diff
@@ -286,6 +286,14 @@ TEST(ELFObjectFileTest, MachineTestForXtensa) {
     checkFormatAndArch(Data, Formats[Idx], Triple::xtensa);
 }
 
+TEST(ELFObjectFileTest, MachineTestForEraVM) {
+  // TODO Should we support both LSB and MSB?
```

Comment:
No. EraVM is BE

Reply:
Looks like other targets generally use endianness information to distinguish between two valid options (such as `mips` vs. `mipsel`) but not to reject invalid combinations - say, i386 usually just assumes "the only possible" little-endian variant (there are [exceptions](https://github.com/matter-labs/era-compiler-llvm/blob/95cb0e966ba7411a9b27eab241b3ff074135cc8f/llvm/include/llvm/Object/ELFObjectFile.h#L1320), though even AMDGPU target seems to not reject big-endian everywhere). Thus, simply dropped this TODO and replaced LSB with MSB in test inputs: df28628823eb21b8d84d0c9338a54547aa60df2e.